### DISCUSSION TITLE
i18n support is not compatible with next export

### DISCUSSION THREAD

**Discussion by TangoPJ** [vinissimus/next-translate#1083] [Q&A] [answered]
Hello:) I have a problem, could somebody help me with advice? 
error i18n support is not compatible with next export. See here for more info on deploying: https://nextjs.org/docs/messages/export-no-custom-routes
Can I build my landing page for production as an SSG?
The error above I'm getting if I run the command yarn build (next build && next export)
<img width="720" alt="image" src="https://github.com/aralroca/next-translate/assets/64611024/4dca5c13-733a-4ca3-b76b-7b3a9daa44b5">

**Comment by aralroca** [comment, marked as answer]
This is normal. `next export` was not supported by Next.js with i18n routing. Now it's deprecated to replace it with `output: export` as the error tells you, but still it's not supported by Next.js team either, here you have more info:

https://nextjs.org/docs/pages/building-your-application/routing/internationalization#how-does-this-work-with-static-generation

What you can do is report it in the Next.js repository because we can't fix it from this library. It has to be the Next.js team who support it.
